src/iomusxml.cpp

```diff
@@ -3834,7 +3834,8 @@ void MusicXmlInput::ReadMusicXmlBeamStart(const pugi::xml_node &node, const pugi
 void MusicXmlInput::ReadMusicXmlTies(
     const pugi::xml_node &node, Layer *layer, Note *note, const std::string &measureNum)
 {
-    for (pugi::xml_node xmlTie : node.children("tied")) {
+    pugi::xpath_node ties = node.select_node("notations[tied]");
+    for (pugi::xml_node xmlTie : ties.node().children("tied")) {
```

Comment:
You need to check that `ties` and `ties.node()` are not null

Reply:
changed approach